crowbar_framework/config/application.rb

```diff
@@ -44,5 +44,20 @@ class Application < Rails::Application
       g.test_framework :rspec, fixture: true
       g.fallbacks[:rspec] = :test_unit
     end
+
+    config.before_initialize do
+      Dotenv.load Rails.root.join("config", "crowbar.env")
+
+      begin
+        Chef::Config.tap do |config|
+          config.node_name ENV["CHEF_NODE_NAME"]
+          config.client_key ENV["CHEF_CLIENT_KEY"]
+          config.chef_server_url ENV["CHEF_SERVER_URL"]
+          config.http_retry_count 3
+        end
+      rescue LoadError => e
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `e`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)